src/hooks/useIsVisible.tsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState, RefObject } from "react";
+
+export function useIsVisible(ref: RefObject<HTMLElement>): boolean {
+    const [isIntersecting, setIntersecting] = useState(false);
```

Comment:
SET in use state is usually the same name as the variable with the prefixed "set".
So, in your case, it should look like this -> setIsIntersecting

Reply:
done